### PR TITLE
test: enabled tests for #1132

### PR DESCRIPTION
**Pull request by ravijayaramappa**
## Details
issue #1132
https://github.com/salesforce/lwc/issues/1132

## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`

If yes, please describe the impact and migration path for existing applications.

## The PR fulfills these requirements:
* Have tests for the proposed changes been added? ✅
* Have you followed [these instructions](../CONTRIBUTING.md#-commit-message-conventions) to clearly describe the issue being fixed or feature enhanced? ✅

### CODE REVIEW

File: packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js [hunk 1]
```diff
@@ -93,8 +92,7 @@ describe('event propagation in simple shadow tree', () => {
         ]);
     });
 
-    // TODO: #1132 - Event.prototype.composedPath() doesn't return the shadow root
-    xit('propagate event from a child element in a document fragment', () => {
+    it('propagate event from a child element in a document fragment', () => {
```
Comment (jodarove) [inline packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js]: @ravijayaramappa this is failing in ie11

Comment (ravijayaramappa) [reply packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js]: @caridy this is regarding event bubbling in a disconnected dom tree.
In IE11, when an event is fired from inside of a document fragment, the event does not bubble up to the fragment. This is the native IE11 behavior.

However in IE11, @lwc/synthetic-shadow patches composed path to include the document fragment. So there is inconsistency here because the composed path says the event will bubble up to the fragment and the event stops just below the fragment.

This is the cause of this test failure. Seems like a corner case. We can write an alternative test case for IE11, to verify that listeners up to the fragment are invoked.
